ci/environment.yml

```diff
@@ -20,8 +20,8 @@ dependencies:
   - scikit-learn
   - scipy
   - shapely
-  - spatialpandas
   - xarray
   - pip:
     - antimeridian
     - pyfma
+    - git+https://github.com/holoviz/spatialpandas.git
```

Comment:
This is until their official release, right?

Reply:
Testing this, awesome work. Gimme a day, at SC..

Reply:
> This is until their official release, right?

Correct

Reply:
Please open an issue if we haven't done so yet in order not to forget this

Reply:
#588